### DISCUSSION TITLE
Plans for account switching?

### DISCUSSION THREAD

**Discussion by Tess-McBoots** [robertodoering/harpy#522] [Ideas]
Hallo! I LOVE the design and functionality of Hapry--are there any plans to add support for multiple accounts?

Thanks!

**Comment by robertodoering** [comment]
Hey, thanks!
Yeah plans exist, here is the issue: https://github.com/robertodoering/harpy/issues/249

**Reply by Tess-McBoots** [reply]
Rock on, thanks!!
